.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,6 +10,15 @@ Before you submit, please search open/closed github issues and discussions since
 **Describe the bug**
 A clear and concise description of what the bug is.
 
+**UI Kit Version Number**
+Please provide the version number you are using.
+
+**Browser**
+Which internet browser are you using?
```

Comment:
It’s important for me to know the operating system and the exact version of both the OS and browser (if possible), since some bugs only occur on certain OS'es.

```suggestion
Please provide the operating system (e.g., Windows 10, macOS Ventura, Ubuntu 22.04) and browser version (e.g., Chrome 116.0, Firefox 117.0) where the issue can be reproduced.
```

Reply:
Good point! Updated here: https://github.com/commercetools/ui-kit/pull/2891/commits/b6339281d700ceb56958ed497c9154c8e6ea2b11

Thank you.